<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrackControllerUI</class>
 <widget class="QMainWindow" name="TrackControllerUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>748</width>
    <height>402</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>550</width>
    <height>400</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>215</red>
        <green>227</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>215</red>
        <green>227</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>215</red>
        <green>227</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>215</red>
        <green>227</green>
        <blue>240</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Track Controller (HW)</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::RightToLeft</enum>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="buttonSave">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buttonBrowse">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Browse</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEditBrowse">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUpload">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Upload PLC File:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="labelCrossingJunction">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Crossing/Junction View:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelPhoto">
          <property name="maximumSize">
           <size>
            <width>440</width>
            <height>250</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap>../../../../../../Documents/TrackController/blueline.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QCheckBox" name="checkGreen">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Green</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkRed">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Red</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBlue">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Blue</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelSelectLine">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Select Line:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3"/>
          </item>
          <item>
           <widget class="Line" name="line_4">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QComboBox" name="comboBoxWayside">
              <property name="layoutDirection">
               <enum>Qt::LeftToRight</enum>
              </property>
              <property name="placeholderText">
               <string>Select</string>
              </property>
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="labelSelectWayside">
              <property name="font">
               <font>
                <family>Times New Roman</family>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Select Wayside:</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="comboBoxCrossing">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="placeholderText">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelCrossing">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Select Crossing:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QComboBox" name="comboBoxJunction">
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="placeholderText">
             <string>Select</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="labelJunction">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>12</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Select Junction:</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="Line" name="line_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
         <widget class="QLabel" name="labelMode">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>20</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Mode:</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QCheckBox" name="checkManual">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Manual</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkAuto">
            <property name="font">
             <font>
              <family>Times New Roman</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Automatic</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <widget class="QLineEdit" name="lineEditOccupiedBlocks">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelOcuppiedBlocks">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Occupied Blocks:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <widget class="QLineEdit" name="lineEditClosedBlocks">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelClosedBlocks">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Closed Blocks:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
